include/tvSettings.h

```diff
@@ -2622,6 +2622,76 @@ tvError_t GetLdimZoneShortCircuitStatus(unsigned char* shortcircuit_zone_list, u
  */
 tvError_t GetNumberOfDimmingZones(unsigned int* number_of_dimming_zones);
 
+/**
+ * @brief Sets 2Point Custom Whitebalance
+ *
+ * This function sets Whitebalance for a current picture mode index, primary video source @n
+ * and primary video format.
+ *
+ * @param[in] color                 - Color type value. Valid value will be a member of ::tvWBColor_t
+ * @param[in] control               - Control index value. Valid values will be a member of ::tvWBControl_t
+ * @param[in] value                 - The WhiteBalance Value to be set.
+ *
+ * @return tvError_t
+ *
+ * @retval tvERROR_NONE                      - Success
+ * @retval tvERROR_INVALID_PARAM             - Input parameter is invalid
+ * @retval tvERROR_INVALID_STATE             - Interface is not initialized
+ * @retval tvERROR_OPERATION_NOT_SUPPORTED   - Operation is not supported
+ * @retval tvERROR_GENERAL                   - Underlying failures - SoC, memory, etc
+ *
+ * @pre TvInit() should be called before calling this API
+ */
+tvError_t SetCustom2PointWhiteBalance(tvWBColor_t color, tvWBControl_t control, int value);
+
+/**
+ * @brief Gets the current Custom WhiteBalance
+ *
+ * This function gets the current WhiteBalance value for the primary video source selected,
+ * primary video format played,picture mode selected,given color and given control value..
```

Comment:
Yes it will return the current Custom WhiteBalance. Which means RGB Gain offset values for the user(custom) color temperature of current source/pqmode/format.